apps/coscrad-frontend/src/utils/generic-components/presenters/image-full-page-width.tsx

```diff
@@ -7,9 +7,14 @@ interface FullImageViewProps {
 }
 
 const StyledFullViewFigure = styled('figure')({
-    width: '100%',
-    display: 'block',
-    margin: '0 0 2% 0',
+    left: '50%',
+    marginLeft: '-50vw',
```

Comment:
I'm not sure if there is a way to calculate these, but if so, we probably want to do that whenever we have a second case where we need to escape via negative margins. But for now, I think this is fine.